cocos/base/CCDirector.cpp

```diff
@@ -534,7 +562,7 @@ void Director::multiplyMatrix(MATRIX_STACK_TYPE type, const Mat4& mat)
     }
     else if(MATRIX_STACK_TYPE::MATRIX_STACK_PROJECTION == type)
     {
-        _projectionMatrixStack.top() *= mat;
+        _projectionMatrixStackList[0].top() *= mat;
```

Comment:
I think we should add comment in `Director::multiplyMatrix` that describe only `_projectionMatrixStackList[0]` is affected.